README.md

```diff
@@ -41,6 +45,7 @@ conda install -c conda-forge mpi4py
 ## Dependencies
 * `numpy`, `astropy`, `scipy`, `mpi4py`, `pyyaml`, `six`, `pyuvdata`
 * optionally `line_profiler` if you want to do profiling (support for profiling is built in)
+* `h5py` is required to support the new `uvh5` data format.
```

Comment:
is it an optional or required dependency?

Reply:
Required for that feature of pyuvdata, but not for pyuvsim.

Reply:
I think you should spell that out a bit more. Maybe something like:

optionally `h5py` to use pyuvdata's `uvh5` file format